core/server/utils/image-size-from-url.js

```diff
@@ -79,9 +79,16 @@ module.exports.getImageSizeFromUrl = function getImageSizeFromUrl(imagePath) {
                             context: imagePath
                         }));
                     }
+                } else if (res.statusCode === 404) {
```

Comment:
I don't understand the change 🤔  

If we switch this error to a 404, the cached image size utility will still log the error to stdout, [see](https://github.com/TryGhost/Ghost/blob/1.5.1/core/server/utils/cached-image-size-from-url.js#L27) . The error (doesn't matter if 500 or 404) only appears once, because the cached utility keeps care of that. So i am wondering how this reduces noise?

Maybe i am overseeing something 🙈  Let me know!